src/Explorer/Panes/DeleteDatabaseConfirmationPanel.test.tsx

```diff
@@ -0,0 +1,79 @@
+jest.mock("../../Common/dataAccess/deleteDatabase");
+jest.mock("../../Shared/Telemetry/TelemetryProcessor");
+import { shallow } from "enzyme";
+import * as ko from "knockout";
+import React from "react";
+import { deleteDatabase } from "../../Common/dataAccess/deleteDatabase";
+import * as ViewModels from "../../Contracts/ViewModels";
+import Explorer from "../Explorer";
+import { DeleteDatabaseConfirmationPanel } from "./DeleteDatabaseConfirmationPanel";
+
+describe("Delete Database Confirmation Pane", () => {
+  describe("Explorer.isLastDatabase() and Explorer.isLastNonEmptyDatabase()", () => {
```

Comment:
Move this block of tests to a new `Explorer.test.ts` file. This is not testing anything related to the panel

Reply:
Moved these test cases in new file `Explorer.test.tsx`